packages/react-components/src/index.ts

```diff
@@ -55,7 +69,32 @@ export type {
   StrokeWidthTokens,
   Theme,
 } from '@fluentui/react-theme';
-export { SSRProvider } from '@fluentui/react-utilities';
+export {
+  getNativeElementProps,
+  getNativeProps,
+  getPartitionedNativeProps,
+  getSlots,
+  resetIdsForTests,
+  resolveShorthand,
+  useId,
+  useIsSSR,
+  useMergedRefs,
+  useMount,
```

Comment:
```suggestion
```

Took a look at this internally too, it's just an effect, there's no point in this either IMO